src/config/types.ts

```diff
@@ -139,6 +139,11 @@ export type Config = {
       beforeNavLinks?: React.ComponentType<any>[]
       afterNavLinks?: React.ComponentType<any>[]
       Nav?: React.ComponentType<any>
+      logout?: {
```

Comment:
looks like this casing is actually right here, but wrong in the docs. Maybe this means the docs just need to be updated?

Reply:
Updated both documentation and code.